django/core/management/templates.py

```diff
@@ -76,6 +77,7 @@ def handle(self, app_or_project, name, target=None, **options):
         else:
             if app_or_project == 'app':
                 self.validate_name(os.path.basename(target), 'directory')
+                app_path = target.replace('/', '.')
```

Comment:
This will work only for a simple use case, e.g.
```
python manage.py startapp test_four ../../main_directory/ticket_30618/ticket_30618/other_apps_3
```
produces `name = '......main_directory.ticket_30618.ticket_30618.other_apps_3'`